Casamari/MNC017.xml

```diff
@@ -0,0 +1,855 @@
+                              <locus target="#142va"/>
+                              <hi rend="rubric">ውዳሴ፡ ዘሐሙሰ፡ ዕፀ</hi><supplied reason="omitted">፡</supplied> እንተ፡ ርእየ፡ ሙሴ፡ በነደ፡ እሳት፡ ውስተ፡ ገዳም፡ ወአዕፁቂሃ፡ 
+                              ኢትውዒ፡ ትመስል፡ <hi rend="rubric">ማርያም፡</hi> ድንግል፡ ዘእንበለ፡ ርኩስ፡ ተሰብአ፡ እምኔሃ፡ ቃለ፡ አብ፡ 
+                           </incipit>
+                           <explicit xml:lang="gez">
+                              <locus target="#144va"/>
+                              ወተወልደ፡ በቤተ፡ ልሔም፡ በከመ፡ ሰበኩ፡ ነቢያት፡ አድኃነነ፡ ወቤዘወነ፡ ወረሰይነ፡ ሕዝቦ፡ ዚአሁ። ሰ፡ ለ፡ ቅ።
+                           </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.4.5">
+                           <locus from="144va" to="145vb"/>
+                           <title type="complete" ref="LIT2509Weddas#Friday"/>
+                           <incipit xml:lang="gez">
+                              <locus target="#144va"/>
+                              <hi rend="rubric">ውዳሴ፡ ዘዓር<supplied reason="omitted">ብ፡</supplied></hi> ቡርክት፡ አንቲ፡ እምአንስት፡ <supplied reason="omitted">፡</supplied>
+                              <hi rend="rubric"><sic>ወቡርክ፡</sic>ወቡርክ፡ ፍሬ፡ ከርሥኪ፡ ኦማርያም<supplied reason="omitted">፡</supplied></hi> ወላዲተ፡ አምላክ፡ ዘእንበለ፡ ርኩስ፡
+                            </incipit>
+                           <explicit xml:lang="gez">
+                              <locus target="#145vb"/>
+                              በእንተዝ፡ ንዌድሰኪ፡ በከመ፡ ገብርኤል፡ መልአክ፡ እንዘ፡ ንብል፡ ቡርክት፡ አንቲ፡  <hi rend="rubric">ማርያም፡</hi> ወቡሩክ፡ ፍሬ፡ ከርሥኪ፡ ተፈሥሒ፡ 
+                              ኦምልዕተ፡ ጸጋ፡ እግዚአብሔር፡ ምስሌኪ፡ ሰ። ለ፡ ቅ።
+                           </explicit>
+                         </msItem>
+                        <msItem xml:id="ms_i1.4.6">
+                           <locus from="145vb" to="146vb"/>
+                           <title type="complete" ref="LIT2509Weddas#Saturday"/>
+                           <incipit xml:lang="gez">
+                              <locus target="#145vb"/>
+                              <hi rend="rubric">ውዳሴ፡ ዘቀዳሚት፡</hi> ንጽሕት፡ ወብርህት፡ ወቅድስት፡ በኵሉ፡ እንተ፡ ሐቀፈቶ፡ ለእግዚእ፡ በእራኃ፡ ወኵሉ፡ ፍጥረት፡ ይትፌሥሑ፡ 
+                              ምስሌሃ፡ እንዘ፡ ይጼርሑ፡ ወይብሉ፡ ሰ።
+                           </incipit>
+                           <explicit xml:lang="gez">
+                              <locus target="#146vb"/>
+                              ዘኢይጸንን፡ ምስማኮሙ፡ ለቅዱሳን፡ ሰአሊ፡ ለነ፡ ኀበ፡ ወልድኪ፡ ኄር፡ መድኃኒነ፡ ይምሐረነ፡ ወይሣሃለነ፡ ይስረይ፡ ኃጢአተነ፡ ሰ። ለ። ቅ።
+                           </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.4.7">
+                           <locus from="146vb" to="148ra"/>
+                           <title type="complete" ref="LIT2509Weddas#Sunday"/>
+                           <incipit xml:lang="gez">
+                              <locus target="#146vb"/>
+                              <hi rend="rubric">ውዳሴ፡ በሰንበተ፡ ክርስቲያን<supplied reason="omitted">፡</supplied></hi> ተሰመይኪ፡ ፍቅርተ፡ ኦቡርክት፡ እምአንስት፡ 
```

Comment:
maybay: supplied **resp="MV"** reason="omitted

I do not (!) mark this up. For the catalogue it doesn't matter, right?

Reply:
No, for the catalogue it doesn't (we are not going to state who is correcting what, that would be pedant). I usually do not mark this up, especially if the correction is very trivial. I've used resp="MV" those (very few) time in which I chose to take the responsibility of a correction (e.g. in MNC-019 the third song of Moses is mistakenly entitled ዳግም፡ again: I marked it up with `<sic resp="MV">ዳግም፡</sic>` because the correction is not immediate.